pkgs/development/compilers/zig/cc.nix

```diff
@@ -16,21 +18,61 @@ wrapCCWith {
 
         nativeBuildInputs = [ makeWrapper ];
 
+        outputs = [
+          "out"
+          "lib"
+        ];
+
         passthru.isZig = true;
         inherit zig;
       }
       ''
         mkdir -p $out/bin
-        for tool in ar cc c++ objcopy; do
+        for tool in cc c++; do
           makeWrapper "$zig/bin/zig" "$out/bin/$tool" \
             --add-flags "$tool" \
             --run "export ZIG_GLOBAL_CACHE_DIR=\$(mktemp -d)"
         done
 
         mv $out/bin/c++ $out/bin/clang++
         mv $out/bin/cc $out/bin/clang
+
+        # Fake lib output needed
+        mkdir -p $lib
```

Comment:
Should this be dropped too?

Reply:
Yes, I missed that before. It's dropped now.

Reply:
Seems like it was dropped in the wrong commit. It's still here in the one that added it.

Reply:
Huh? I checked the bland and I don't see it on GitHub

Reply:
https://github.com/NixOS/nixpkgs/pull/331011/commits/7a14c5455ed34d37f33a62e8ae5a15c10a5bf6c7

Reply:
Fixed